### PR TITLE
Fix the istio-proxy-cfg script

### PR DESCRIPTION
**Pull request by robcresswell**
Update the parameter parsing in `istio-proxy-cfg` to pull the correct values and function correctly.

### CODE REVIEW

**Comment by robcresswell** [comment]
I signed it!

File: bin/istio-proxy-cfg [hunk 1]
```diff
@@ -69,7 +69,7 @@ function check_endof_cmdline {
 
 function get_pilot_url {
     if [[ -z $PILOT_URL ]]; then
-        pilot_ipport=(`kubectl get svc -n ${ISTIO_NAMESPACE} | grep ${ISTIO_PILOT} | awk '{print $2,$4}'`)
+        pilot_ipport=(`kubectl get svc -n ${ISTIO_NAMESPACE} | grep ${ISTIO_PILOT} | awk '{print $3,$5}'`)
```
Comment (baodongli) [inline bin/istio-proxy-cfg]: This is great. I'd like to see a follow-up change to make it agnostic to k8s version.

**Pull request merged**